parse_pom_sequence.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+#
+# Common utilities used by tools for automating tagging and release
+#
+
+#We want any failures to be fatal
+# Depends: maven
+
+set -e
+
+if ! `which mvn 1>/dev/null 2>&1` ; then
+  echo "Maven dependency not located"
+  exit 2
+fi
+
+# overwrite versionfile to pom.xml
+VERSIONFILE=pom.xml
+if [ ! -f $VERSIONFILE ]; then
+  echo "Expected $VERSIONFILE not found"
+  exit 2
+fi
+
+
+# overwrite parse_cmake to target pom files
+# expect ex: 7.6.2-0-SNAPSHOT, 7.6.2-1 (gold release), etc
+# Major.Minor.Point- Sequence if gold, Maturity if rc
+function parse_cmake()
+{
+  HPCC_PROJECT=$(mvn help:evaluate -Dexpression=project.artifactId -q -DforceStdout)
```

Comment:
this mvn command will act on the local pom? It might be worthwhile to log the fully qualified name of the effective pom file.

Reply:
Added in a catch which will error out if we're using the versioning system without a <sequence>

Reply:
based on Richard's workflow, we'll be in the root directory of the project where the pom is located.